gpt4all-chat/qml/Theme.qml

```diff
@@ -644,13 +651,21 @@ QtObject {
 
     property color collectionsButtonBackground: {
         switch (MySettings.chatTheme) {
+            case "LegacyDark":
```

Comment:
Let's leave this out. If all three cases return the same thing, then the way I'm doing it now with just a 'default' is the way to go.